text_formatter_test.go

```diff
@@ -443,7 +444,11 @@ func TestTextFormatterIsColored(t *testing.T) {
 				os.Setenv("CLICOLOR_FORCE", val.clicolorForceVal)
 			}
 			res := tf.isColored()
-			assert.Equal(subT, val.expectedResult, res)
+			if runtime.GOOS == "windows" {
```

Comment:
I think we need to updated the test  for windows environment where isColored always return false except when ForceColor is set to true.